### PR TITLE
feat (replays/33273): added project column to the replays screen

### PR DESCRIPTION
**Pull request by Danondso**
# Description
#33273 
- Added the project column to the replays screen

# Screenshots
image
image

# Known Gotchas
- none

### CODE REVIEW

File: static/app/views/replays/replays.tsx [hunk 1]
```diff
@@ -61,7 +63,7 @@ class Replays extends React.Component<Props> {
       <React.Fragment key={replay.id}>
         <Link
           to={`/organizations/${organization.slug}/replays/${generateEventSlug({
-            project: replay['project.name'],
```
Comment (ryan953) [inline static/app/views/replays/replays.tsx:64]: good change, looks like `project.name` wasn't part of `static/app/views/replays/types.tsx` before either.

File: static/app/views/replays/replays.tsx [hunk 2]
```diff
@@ -79,6 +81,18 @@ class Replays extends React.Component<Props> {
             displayEmail={replay.url?.split('?')[0] || ''}
           />
         </Link>
+        <Projects orgId={organization.slug} slugs={[replay.project]}>
```
[the remark this answers is not in the document]
Comment (Danondso) [reply static/app/views/replays/replays.tsx:84]: I'll find out here and adjust accordingly.

[the remark this answers is not in the document]
Comment (Danondso) [reply static/app/views/replays/replays.tsx:84]: Doesn't seem to be the case as far as I can tell. <Projects /> leverages a reflux store that stores projects that have already been fetched along with managing calls to the API for getting any projects that don't yet exist in the store.

That call to `/projects` there is the only one that occurs for loading up the replays screens. A refresh will cause it to pull again but visiting other pages and coming back doesn't trigger the call.
image

File: static/app/views/replays/types.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export type Replay = {
```
Comment (ryan953) [inline static/app/views/replays/types.tsx:1]: More TS shenanigans.. .
`projectID` isn't fetched, and `replayId` seems to be empty / not used any of the time.

**Review by ryan953 — APPROVED** [review]

File: static/app/views/replays/types.tsx (hunk 3 above)
Comment (Danondso) [reply static/app/views/replays/types.tsx:1]: Oh good catch I'll get those removed.

Comment (Danondso) [reply static/app/views/replays/types.tsx:1]: Alright removed those.

**Comment by Danondso** [comment]
I goofed on my merging and wrecked the diff, addressing that before opening this again.

**Pull request closed without merging**